frontend/src/css/collapsing-item.css

```diff
@@ -0,0 +1,25 @@
+/*
+collapsing-item.css is used for the CollapsingItemContainer.jsx styles that cannot be added inside a const
+*/
+
+.collapsing-item-container:hover .expanded-icon {
+  color: white !important;
```

Comment:
do you need the !importants? this should override any other css, since it is a different css file. I thought I needed some in previous PRs and realized that I didn't

Reply:
Yes I do. Otherwise, when you put your mouse over the button, the arrow icon is the same color as the button.

Reply:
Sounds good! I just know that I overuse it

Reply:
It looks like this is because you've set the color in your react, which overrides it. Maybe take it out of there, or use an `id` instead of a class to set the style so it's more specific.